lib/modules/orangered.js

```diff
@@ -130,16 +130,27 @@ export function updateFromPage(doc?: HTMLElement) {
 	setUnreadCount(getUnreadCount(doc));
 }
 
-const _setUnreadCount = multicast(count => {
-	count = count || 0;
-
-	updateFaviconBadge(count);
-	updateTitle(count);
-	updateInboxElements(count);
-	updateMailCountElements(count);
-
-	if (count > 0) {
-		window.dispatchEvent(new Event('orangered'));
+const _setUnreadCount = multicast(data => {
```

Comment:
The callers of this function still use `_setUnreadCount(count)`, they should probably be renamed (can't comment on those lines)